Scutea Emilia/Lab4 3.10/main.c

```diff
@@ -0,0 +1,76 @@
+#include <stdio.h>
+#include <stdlib.h>
+int a;
+int *nr;
+void nr_bancnote (int *nr, int a)
```

Comment:
Quite a lot of redundant code here. You could iterate through an array which contains all those values and then apply the same while loop on every value of that array. You'll have to refactor this